includes/PatternBuilder/Property/DrupalPatternBuilderValueProperty.php

```diff
@@ -73,7 +84,34 @@ public function render() {
    * {@inheritdoc}
    */
   public function prepareRender() {
-    return $this->data;
+    $values = new \stdClass();
```

Comment:
prepareRender() and values() duplicate code. Create a private method with that duped code in pass in a method exists argument. Have the two methods call the private method with the proper argument.